src/test/java/gim/testutil/ExerciseBuilder.java

```diff
@@ -59,10 +60,10 @@ public ExerciseBuilder withName(String name) {
     }
 
     /**
-     * Parses the {@code tags} into a {@code Set<Tag>} and set it to the {@code Exercise} that we are building.
+     * Parses the {@code dates} and set it to the {@code Exercise} that we are building.
      */
-    public ExerciseBuilder withTags(String ... tags) {
-        this.tags = SampleDataUtil.getTagSet(tags);
+    public ExerciseBuilder withDates(String date) {
```

Comment:
Maybe refactor the method to withDate instead?

Reply:
The necessary changes have been made.